pkg/task/generictaskhandler.go

```diff
@@ -457,10 +457,14 @@ func applyResourceMutations(ctx context.Context, draft repository.PackageRevisio
 		}
 		if taskResult != nil && task.Type == api.TaskTypeEval {
 			renderStatus = taskResult.RenderStatus
+			if err != nil {
+				klog.Error(err)
+				err = fmt.Errorf("%w\n\n%s\n%s\n%s", err, "Error occurred rendering package in kpt function pipeline.", "Package has NOT been pushed to remote.", "Please fix package locally (modify until 'kpt fn render' succeeds) and retry.")
```

Comment:
Should the error refer to "evaluating"? 
ie. task.Type == api.TaskTypeEval

Reply:
i am not sure what is right or wrong in this situation. 
i just think its poor naming consistency and the terms seem to be used interchangeably in the code.
i have no issue rewording form rendering to evaluating if that is what is the correct term.